packages/databricks-vscode/resources/whats-new/2.4.md

```diff
@@ -0,0 +1,35 @@
+<img src="./1.2/databricks-logo.png" alt="databricks-logo" width="100"/>
+
+# Databricks Extension v2.4.0
+
+## What's new?
+
+-   [Databricks Asset Bundles support](#dabs)
+-   [Bundle configuration](#dabs-config)
+-   [No environment variables in terminals](#no-env)
```

Comment:
This last one doesn't seem like whats-new item to me (it's a behavior change), so maybe it shouldn't be in here? Or change the title of this file to "What's changed?"

Reply:
It's a new behaviour, I think it still fits here